authors/apps/authentication/views.py

```diff
@@ -72,3 +89,63 @@ def update(self, request, *args, **kwargs):
         serializer.save()
 
         return Response(serializer.data, status=status.HTTP_200_OK)
+      
+class ResetPasswordAPIView(generics.CreateAPIView):
+    permission_classes = (AllowAny,)
+    # authentication_classes = (JWTAuthentication,)
+    renderer_classes = (UserJSONRenderer,)
+    serializer_class = LoginSerializer
+
+    def post(self, request):
+        #get user data
+        user_data = request.data['user']['email']
+
+        token = generate_reset_token(user_data)
+        try:
+            sg = sendgrid.SendGridAPIClient(apikey = SENDGRID_API_KEY)
```

Comment:
@emukungu, sync with @johnkegz to find out if he went ahead with the SendGrid way for sending emails

Reply:
changed to smtp.gmail, similar to what John is using